### ISSUE TITLE
Incorrect error message in the section payment

### ISSUE THREAD

**Issue by juraj1000** [PrestaShop/PrestaShop#19740]
#### Describe the bug
There is an incorrect message in the section Payment if a merchant wants to edit an order item in BO.

#### Expected behavior
No message should be dispalyed.

#### Steps to Reproduce
1. Go to BO > Orders > Orders > select order and display detail screen
2. Scroll down to the section Products and click on button Edit
3. Click on button Update
4. Confirm question Are you sure? with OK
5. See red warning message in the section Payments "Warning €54.40 paid instead of €54.40"

**Screenshots**
![image](https://user-images.githubusercontent.com/33311857/84599994-71fcfa80-ae76-11ea-841e-50f9bb309842.png)

#### Additional information
* PrestaShop version: 1.7.6.5
* PHP version: 7.1.13

**Comment by matthieu-rolland** [comment]
Hi @juraj1000 , could you please give us more information about the payment method being used ? A capture of the whole page would help, thanx !

**Comment by juraj1000** [comment]
Hello,
there is used payment method bank transfer

Here is a new order with the same issue
![image](https://user-images.githubusercontent.com/33311857/84662794-8f849f80-af1c-11ea-9678-f8f9b38026d0.png)
![image](https://user-images.githubusercontent.com/33311857/84662824-9c08f800-af1c-11ea-8fb8-db5dd1425554.png)

**Comment by florine2623** [comment]
@juraj1000 ,
What exactly are you changing on the product section ?
Price ? Quantity ? More or less ?

**Comment by juraj1000** [comment]
Hello,
don't change anything, only click the button Update. See steps how to reproduce it in bug description.

**Comment by florine2623** [comment]
What is the order status when you try to update the price ?

**Comment by juraj1000** [comment]
There is Payment accepted, you can see it on the screen which I posted here in previous reply

**Comment by florine2623** [comment]
Ok, I did manage to reproduce the issue with PS 1.7.6.5 :
![Capture d’écran 2020-06-19 à 16 00 53](https://user-images.githubusercontent.com/16019289/85140838-24053f80-b246-11ea-89c6-3b5db171c8a7.png)
However in PS 1.7.7.0, the bug has been fixed. The whole order page has been changed.
I won't be able to click Update in the product section with no change to it. There is a 'Save' button when changes are made and 'Cancel' button when you have no change to make.
![Capture d’écran 2020-06-19 à 16 02 23](https://user-images.githubusercontent.com/16019289/85141045-75153380-b246-11ea-9767-71277e321276.png)
Thanks!
